src/pages/Randomiser.jsx

```diff
@@ -489,6 +501,8 @@ export const Randomiser = () => {
                             id={o.id}
                             checked={state[o.id]}
                             defaultValue={state[o.id]}
+                            {...(o.min >= 0 ? { min: o.min } : {})}
+                            {...(o.max ? { max: o.max } : {})}
```

Comment:
If either of these props are `undefined` they won't be rendered, so can simplify this instead, like:

```suggestion
                           min: {o.min ?? undefined}
                           max: {o.max ?? undefined}
```